net/gce.sh

```diff
@@ -799,36 +805,55 @@ info)
     printf "  %-16s | %-15s | %-15s | %s\n" "$nodeType" "$ip" "$ipPrivate" "$zone"
   }
 
-  printNode "Node Type" "Public IP" "Private IP" "Zone"
-  echo "-------------------+-----------------+-----------------+--------------"
+  if ! $evalInfo; then
+    printNode "Node Type" "Public IP" "Private IP" "Zone"
+    echo "-------------------+-----------------+-----------------+--------------"
+  fi
+
   nodeType=bootstrap-leader
   for i in $(seq 0 $(( ${#validatorIpList[@]} - 1)) ); do
     ipAddress=${validatorIpList[$i]}
     ipAddressPrivate=${validatorIpListPrivate[$i]}
     zone=${validatorIpListZone[$i]}
-    printNode $nodeType "$ipAddress" "$ipAddressPrivate" "$zone"
+    if $evalInfo; then
+      echo "NET_VALIDATOR${i}_IP=$ipAddress"
```

Comment:
Looks like the bootstrap leader will be `NET_VALIDATOR0_IP`, intentional?

Reply:
yeah intentional